java/src/main/java/vowpalWabbit/learner/VWMulticlassLearner.java

```diff
@@ -13,4 +13,24 @@
 
     @Override
     protected native int predictMultiline(String[] example, boolean learn, long nativePointer);
+
+    protected native float[] rawPredict(String example, boolean learn, long nativePointer);
+
+    /**
+     * Get raw prediction output.
+     *
+     * @param example a single vw example string
+     * @return Raw prediction
+     */
```

Comment:
I don't think this code should go here.  From what I understand about VW, the action of getting a raw prediction is applicable to a wider variety of cases than just multi class prediction.  @JohnLangford can you comment on this.  If this is the case then we'd want to put this code higher up in the chain and allow more learners to take advantage of it.  If not, then this seems fine.  @deaktator what do you think?